sky/backends/backend_utils.py

```diff
@@ -94,8 +97,8 @@
 # Mapping from reserved cluster names to the corresponding group name (logging
 # purpose).
 # NOTE: each group can only have one reserved cluster name for now.
-SKY_RESERVED_CLUSTER_NAMES = {
-    spot_lib.SPOT_CONTROLLER_NAME: 'Managed spot controller'
+SKY_RESERVED_CLUSTER_NAMES: Dict[str, str] = {
+    spot_lib.SPOT_CONTROLLER_NAME: 'Managed spot controller'  # type: ignore
```

Comment:
This `# type: ignore` seems not required - mypy passes without this too

Reply:
Oops, good catch! Removed it. Thanks!